assets/pages/paper-subscription-landing/components/content.jsx

```diff
@@ -0,0 +1,146 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import ProductPageContentBlock from 'components/productPage/productPageContentBlock/productPageContentBlock';
+import ProductPageContentBlockDivider from 'components/productPage/productPageContentBlock/productPageContentBlockDivider';
+import ProductPageTextBlock, { sansParagraphClassName } from 'components/productPage/productPageTextBlock/productPageTextBlock';
+import ProductPageTextBlockList from 'components/productPage/productPageTextBlock/productPageTextBlockList';
+import SvgInfo from 'components/svgs/information';
+import { paperSubsUrl } from 'helpers/routes';
+
+import { type State } from '../paperSubscriptionLandingPageReducer';
+import { setTab, type TabActions } from '../paperSubscriptionLandingPageActions';
+
+import Form from './form';
+
+import './content.scss';
+
+type PropTypes = {|
+  selectedTab: number,
+  setTabAction: typeof setTab
+|};
+
+// ----- Render ----- //
+
+class Content extends Component<PropTypes> {
+
+  componentDidUpdate({ selectedTab }) {
+    if (selectedTab !== this.props.selectedTab && this.tabRef) {
+      // $FlowIgnore
```

Comment:
What error is this ignoring?

Reply:
`focusOptions` which is not part of the flow spec yet but is [defined and documented](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus) by browser vendors and will just be ignored in browsers that don't support it